### DISCUSSION TITLE
Discharge Capacity Doesn't Change?

### DISCUSSION THREAD

**Discussion by wass-cloud** [pybamm-team/PyBaMM#3122] [Q&A]
When I change the Nominal Capacity parameter to a value other than 5, why does my discharge capacity not change from 5 either?

**Comment by brosaplanella** [comment]
See https://github.com/pybamm-team/PyBaMM/discussions/3120

**Reply by wass-cloud** [reply]
So am I unable to change the real capacity of the battery in PyBaMM?

**Reply by brosaplanella** [reply]
You can change other parameters that affect the capacity (e.g. maximum concentration, electrode thickness, active material volume fraction...)

**Reply by wass-cloud** [reply]
I see, thank you.

**Comment by georgemalick412** [comment]
As a user, it's puzzling that changing the Nominal Capacity parameter doesn't affect the discharge capacity. It seems that there may be another factor or setting influencing the discharge [Capacity](https://weldingtrends.com/gorilla-glue-weight-capacity/) value. It could be worth exploring other parameters or configurations related to the discharge capacity to understand why it remains constant at 5 regardless of the Nominal Capacity change.

**Comment by marcchane412** [comment]
As a user, it's puzzling that changing the Nominal Capacity parameter doesn't result in a corresponding change in the discharge capacity. It could indicate that there are other factors influencing the discharge capacity value. It might be worth exploring additional parameters or settings related to the discharge capacity to understand the [Winstrol Benefici](https://fitexpt.com/winstrol-per-le-donne-benefici-ed-effetti-collaterali/) why it remains constant at 5, regardless of the Nominal Capacity change. Consulting a technical expert or reviewing the documentation could provide further insights.

**Reply by ben-kenney** [reply]
It makes sense. The pybamm model is like a "digital twin" of a real cell. If you have an 18650 cell that you're testing, how are you going to magically get higher capacity? The only way to get higher capacity (assuming chemistry and microstructure is the same) is to make the cell bigger, like moving from an 18650 to a 21700 for example. In pybamm you can do exactly this: increase the length and height parameters and you'll get a higher capacity because the active area has increased.
